scripts/in_container/run_tmux.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+if [[ ${START_AIRFLOW:="false"} == "true" ]]; then
+
+    ln -s "${IN_CONTAINER_DIR}/stop_tmux_airflow.sh" stop_airflow.sh
```

Comment:
It would be great to print information about the stop_airlfow.sh command. I find it super useful but it's hard to discover. I think the best way is to print it in the terminal that you are dropped at :) 

"You can use 'stop_tmux_airflow.sh' to kill the whole session"

Reply:
added this too. I had to create a new script file, but at the end it worked.